README.md

```diff
@@ -10,67 +10,77 @@ This is the development home for Cyberduck, a libre file transfer client for mac
 
 Make sure to subscribe to the [localization mailing list](http://lists.cyberduck.ch/mailman/listinfo/cyberduck-localization).
 
 ## Support & Documentation
+
 For general help about using Cyberduck, please refer to the [documentation](https://docs.cyberduck.io).
 
 ## Snapshot and Beta builds
-These are nightly snapshot builds from the current development trunk featuring the latest bugfixes and enhancements. Be warned, tough, these builds are potentially unstable and experimental. You can also switch to beta or snapshot builds in _Preferences → Update.
+
+These are nightly snapshot builds from the current development trunk featuring the latest bug fixes and enhancements. Be warned, though, these builds are potentially unstable and experimental. You can also switch to beta or snapshot builds in _Preferences → Update.
 
 ## Prerequisites
 
-- Java 11 SDK or later
-- Apache Ant 1.10.1 or later
-- Apache Maven 3.5 or later
+* Java 11 SDK or later
+* Apache Ant 1.10.1 or later
+* Apache Maven 3.5 or later
 
 ### macOS
 
-- [Xcode 12](https://developer.apple.com/xcode/download/) or later
+* [Xcode 12](https://developer.apple.com/xcode/download/) or later
 
 ### Windows
 
-- Visual Studio 2017 or later
-  - `.NET Desktop development`-Workload
-  - Windows SDK (10.0.14393.0)
-  - [MSBuild Community Tasks](https://github.com/loresoft/msbuildtasks)
-- [Bonjour SDK for Windows](https://developer.apple.com/downloads/index.action?q=Bonjour%20SDK%20for%20Windows)
+* Visual Studio 2017 or later
+  * `.NET Desktop development`-Workload
+  * Windows SDK (10.0.14393.0)
+  * [MSBuild Community Tasks](https://github.com/loresoft/msbuildtasks)
+* [Bonjour SDK for Windows](https://developer.apple.com/downloads/index.action?q=Bonjour%20SDK%20for%20Windows)
 
 #### Chocolatey
-```
+
+```sh
 choco install adoptopenjdk15 maven bonjour -y
 choco install visualstudio2019buildtools -y
 choco install wixtoolset -y
 choco install visualstudio2019-workload-manageddesktopbuildtools --params "--add Microsoft.Net.Component.4.7.TargetingPack" -y
 choco install visualstudio2019-workload-netcorebuildtools -y
 choco install visualstudio2019-workload-vctools --params "--add Microsoft.VisualStudio.Component.Windows10SDK.17763 --add Microsoft.VisualStudio.Component.VC.v141.x86.x64" -y
 ```
+
 Restart your machine after installing these components.
 
 Additional `%PATH%`:
+
 * `%ProgramFiles(x86)%\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin`
 * `%ProgramFiles(x86)%\Windows Kits\10\bin\10.0.17763.0\x64`
 
 ## Building
 
-Run `mvn verify -DskipTests -DskipSign` to build without running any tests and skip codesign. Find build artifacts in 
- * `osx/target/Cyberduck.app`
- * `windows/target/Cyberduck.exe`
+Run `mvn verify -DskipTests -DskipSign` to build without running any tests and skip codesign. Find build artifacts in
+
+* `osx/target/Cyberduck.app`
+* `windows/target/Cyberduck.exe`
+
+Run with `-pyinstaller` to build installer packages with build artifacts
```

Comment:
There is a new typo introduced.

Reply:
I was not familiar. It has now been reverted.